website/package.json

```diff
@@ -1,17 +1,21 @@
 {
   "license": "MIT",
   "scripts": {
-    "build": "docusaurus-build",
+    "build": "webpack -p && docusaurus-build",
     "publish-gh-pages": "docusaurus-publish",
     "start": "docusaurus-start",
```

Comment:
What about changing this to `docusaurus-start & webpack --watch` (just one `&`) to run them in parallel?

Reply:
https://github.com/kimmobrunfeldt/concurrently